### PR TITLE
CLI update for token, configuration support and api changes 

### PR DESCRIPTION
**Pull request by joseph-v**
**What this PR does / why we need it**:
This PR update the orchestration CLI with support for token generation, configurations from commandline, current apis, bug fixes etc.

### CODE REVIEW

File: orchestration/cli/Readme.txt [hunk 1]
```diff
@@ -6,13 +6,45 @@ Steps to run CLI:
...
+    ./orchctl instance delete --id 2723347b-9af8-451a-a7f8-62d40b10ad6f
+    ./orchctl instance run --data '{"service_id":"08e8a8a3-7a78-43d3-9ab1-45fe7a60d4eb","action":"opensds.provision-volume","name":"Volume Provision name","description":"Volume Provision description","user_id":"558057c4256545bd8a307c37464003c9","parameters":{"ip_addr":"127.0.0.1","port":"50040","tenant_id":"94b280022d0c4401bcf3b0ea85870519","size":1,"name":"test"}}'
+
+positional arguments:
+  {service,instance,workflow,task,info}
+    service             Registers/Shows services in Orchestrator
+    instance            Executes/Shows/Deletes workflow instances
+    workflow            Shows workflow definitions in Orchestrator
+    task                Status of the task in Orchestrator
+    info                Shows configs, tenant_id, user_id, token of Orchestrator
```
Comment (wisererik) [inline orchestration/cli/Readme.txt]: does it mean the sensitive info is displayed here? if that case, please remove it (document and code).

Comment (joseph-v) [reply orchestration/cli/Readme.txt]: If CLI User can provide 'username' and 'password', he can get corresponding tenant_id, user_id, token etc. Do you suggest to remove this command?

Comment (wisererik) [reply orchestration/cli/Readme.txt]: yes

Comment (joseph-v) [reply orchestration/cli/Readme.txt]: Done

**Pull request merged**